Dockerfile

```diff
@@ -1,39 +1,62 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
 ARG ARCH="amd64"
 ARG OS="linux"
 ARG VER="0.22.2"
 ARG PKG="alertmanager"
 ARG SRC="${PKG}-${VER}.${OS}-${ARCH}"
+ARG UID="nobody"
+ARG GID="nobody"
 
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
-
-LABEL	ORG="Armedia LLC" \
-		APP="Prometheus Alert Manager" \
-		VERSION="${VER}" \
-		IMAGE_SOURCE="https://github.com/ArkCase/ark_prometheus_alertman" \
-		MAINTAINER="Armedia LLC"
+LABEL   ORG="Armedia LLC" \
+        APP="Prometheus Alert Manager" \
+        VERSION="${VER}" \
+        IMAGE_SOURCE="https://github.com/ArkCase/ark_prometheus_alertman" \
+        MAINTAINER="Armedia LLC"
```

Comment:
This is not consistent with your change in ark_prometheus...

Reply:
You're right - something's off ... you're not seeing the latest versions. I'm positive I committed and pushed those yesterday.